pynbody/gravity/__init__.py

```diff
@@ -1,5 +1 @@
-import imp
-import numpy as np
-
 from . import calc
-imp.reload(calc)
```

Comment:
The `imp` library has been deprecated in favour of `importlib`, but I couldn't find any reason why we were reloading the .so files?